ovssubnet/common.go

```diff
@@ -336,6 +343,15 @@ func (oc *OvsController) StartNode(sync, skipsetup bool) error {
 	}
 	go oc.watchCluster()
 
+	services, err := oc.subnetRegistry.GetServices()
+	if err != nil {
+		log.Errorf("Could not fetch existing services: %v", err)
+	}
+	for _, svc := range *services {
+		oc.flowController.AddServiceOFRules(oc.VNIDMap[svc.Namespace], svc.IP, svc.Protocol, svc.Port)
+	}
+	go oc.watchServices()
+
```

Comment:
Valid only for multi-tenant flow controller otherwise VNIDMap won't be populated.